### ISSUE TITLE
Catalogue - Product Page - Accessibility Issues

### ISSUE THREAD

**Issue by JamesONeill-DoF** [dof-dss/architecture-catalogue#96]
The "Get Started", "Support", "Features", and "Sign In" button text lacks sufficient contrast against the button background colour.  (Tested using the Wave tool, against WCAG 2.1 AA)
